ui-poc/pages/Friends.py

```
@@ -0,0 +1,350 @@
+                payer_id = expense.get('createdBy')
+                total_amount = expense.get('amount', 0)
+                
+                # Find splits for current user and friend
+                current_user_split = next((s for s in splits if s.get('userId') == current_user_id), None)
+                friend_split = next((s for s in splits if s.get('userId') == friend_user_id), None)
+                
+                if current_user_split or friend_split:
+                    current_user_owes = current_user_split.get('amount', 0) if current_user_split else 0
+                    friend_owes = friend_split.get('amount', 0) if friend_split else 0
+                    
+                    current_user_paid = total_amount if payer_id == current_user_id else 0
```

Ruff (0.11.9)

Local variable `current_user_paid` is assigned to but never used

Remove assignment to unused variable `current_user_paid`

(F841)